zebra/zebra_vty.c

```diff
@@ -942,6 +1112,7 @@ DEFPY(ip_route_vrf,
 	  tag (1-4294967295)                           \
 	  |(1-255)$distance                            \
 	  |label WORD                                  \
+	  |table (1-4294967295)                        \
```

Comment:
Again this should be a separate defun.

Reply:
I had done it for vrf node commands, but was thinking that it was ok for default vrf. so I updated it.